controllers/certificaterequest/issue_certificate.go

```diff
@@ -228,7 +222,14 @@ func (r *CertificateRequestReconciler) IssueCertificate(reqLogger logr.Logger, c
 	return nil
 }
 
-func (r *CertificateRequestReconciler) FindZoneIDForChallenge(namespace string) (string, error) {
+func (r *CertificateRequestReconciler) FindZoneIDForChallenge(namespace string, dnsClient cClient.Client) (string, error) {
+	if fedramp {
+		fedrampZoneid, err := dnsClient.GetFedrampHostedZoneID(fedrampHostedZoneID)
```

Comment:
where does the argument here come from?

Reply:
cClient We imported the cClient on the file and the fedramp logic, we took from the route53.go file